sys/riscv/include/_align.h

```diff
@@ -38,7 +38,7 @@
  * for all data types (int, long, ...).   The result is unsigned int
  * and must be cast to any desired pointer type.
  */
-#define	_ALIGNBYTES	(sizeof(long long) - 1)
-#define	_ALIGN(p)	(((u_long)(p) + _ALIGNBYTES) & ~_ALIGNBYTES)
+#define	_ALIGNBYTES	(_Alignof(__max_align_t) - 1)
```

Comment:
I've pushed a different version using sizeof(void *) matching mips. I think that's what's meant here.